modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -0,0 +1,727 @@
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "neo4j" | "standard" | []      | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | []           |
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+=== Filter the listed databases
+
+It is also possible to filter and sort the results by using `YIELD`, `ORDER BY`, and `WHERE`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES YIELD name, currentStatus, requestedStatus
+ORDER BY currentStatus
+WHERE name CONTAINS 'o'
+----
+
+In this example:
+
+* The number of columns returned has been reduced with the `YIELD` clause.
+* The order of the returned columns has been changed.
+* The results have been filtered to only show database names containing `'o'`.
+* The results are ordered by the `currentStatus` column using `ORDER BY`.
+
+It is also possible to use `SKIP` and `LIMIT` to paginate the results.
+
+
+.Result
+[role="queryresult"]
+----
++--------------------------------------------+
+| name     | currentStatus | requestedStatus |
++--------------------------------------------+
+| "movies" | "online"      | "online"        |
+| "neo4j"  | "online"      | "online"        |
++--------------------------------------------+
+----
+
+[NOTE]
+====
+Note that for failed databases, the `currentStatus` and `requestedStatus` are different.
+This often implies an error, but **that is not always the case**.
+For example, a database may take a while to transition from `offline` to `online` due to a performing recovery.
+Or, during normal operation, a database's `currentStatus` may be transiently different from its `requestedStatus` due to a necessary automatic process, such as one Neo4j instance copying store files from another.
+The possible statuses are `initial`, `online`, `offline`, `store copying` and `unknown`.
+====
+
+
+[role=enterprise-edition not-on-aura]
+[[manage-databases-create]]
+== Create databases
+
+Databases can be created using `CREATE DATABASE`.
+
+[NOTE]
+====
+Having dots (`.`) in the database names is not recommended.
+This is due to the difficulty of determining if a dot is part of the database name or a delimiter for a database alias in a composite database.
+====
+
+.Query
+[source, cypher]
+----
+CREATE DATABASE customers
+----
+
+When a database has been created, it will show up in the listing provided by the command `SHOW DATABASES`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES YIELD name
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+----
++-------------+
+| name        |
++-------------+
+| "customers" |
+| "movies"    |
+| "neo4j"     |
+| "system"    |
++-------------+
+----
+
```

Comment:
was the cluster topology section not moved yet or missed in the move? cause it feels really easy to miss it later on if it was intentionally left out for now

Reply:
It was purposefully left out. We will handle everything clustering related in pt. 2 of this project (migrating server management and access-control), and have made a note not to forget these parts.